src/Microsoft.AspNet.SignalR.Core/Hubs/Extensions/HubTypeExtensions.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace Microsoft.AspNet.SignalR.Hubs
 {
-    internal static class HubTypeExtensions
```

Comment:
We can't really make these public as extension methods on `Type`. Unless there's an extremely good reason, we try to avoid attaching public extension methods on public BCL types like this because they appear for everyone using the type if they happen to have our namespace imported. (If we were writing these helpers now, we wouldn't have made them extension methods at all, even though they are `internal` 😉).

Can you either leave these as `internal` (and just copy the code for your own extensions) or convert them to standard static methods (which would require changing all the call-sites unfortunately).

Reply:
Git your point. It is not that important after all. I have discarded this change during rebase.